test_conformance/extensions/cl_khr_command_buffer/command_buffer_printf.cpp

```diff
@@ -274,7 +274,7 @@ struct CommandBufferPrintfTest : public BasicCommandBufferTest
                                      &pattern[0], 0, nullptr, nullptr);
         test_error(error, "clEnqueueWriteBuffer failed");
 
-        cl_int offset[] = { 0, pattern.size() - 1 };
+        size_t offset[] = { 0, pattern.size() - 1 };
```

Comment:
Can you please check that this change is correct?  I'm nervous that going from a 32-bit type (`cl_int`) to a possibly 64-bit type (`size_t`) may require some additional changes.

Reply:
`offset` is used in 3 locations in this function, two are trivial loop comparisons against another `size_t`, so those are fine. The other, main one, is a call to `clEnqueueWriteBuffer` which expects a `void*` and the size in bytes. The call is already using `sizeof` so I don't see why this would fail regardless of the data type and/or architecture.